README.md

```diff
@@ -37,6 +37,13 @@ tags: ['flapper', 'notify']
 
 
 # If omitted, defaults to system's hostname
+# Can be list like this
+# host:
+#   - "host1.example.com"
+#   - "host2.example.com"
+# or this:
+# host: ["host1.example.com", "host2.example.com"]
```

Comment:
The 'host' option here is for setting the host field of the event (ie, source), not the destination Riemann server.